frontend/src/__mocks__/mockModelRegistryService.ts

```diff
@@ -3,17 +3,26 @@ import { ServiceKind } from '~/k8sTypes';
 type MockServiceType = {
   name?: string;
   namespace?: string;
+  description?: string;
+  serverUrl?: string;
 };
 
 export const mockModelRegistryService = ({
   name = 'modelregistry-sample',
   namespace = 'odh-model-registries',
+  description = 'Model registry description',
+  serverUrl = 'modelregistry-sample-rest.apps.kserve-pm.dev.datahub.redhat.com:443',
```

Comment:
oops missed this... small nit: this needn't be the same URL. Can this be something generic or are you validating this somewhere?

Reply:
@manaswinidas I'm not sure exactly what you mean here, are you saying this shouldn't be a specific real cluster URL and instead should be something like `'sample.url.com:443'`?